src/pages/Home.js

```diff
@@ -86,6 +86,19 @@ class Home extends Component {
       margin: '2rem auto'
     };
 
+    const buttonStyleBig = {
+      borderRadius: '5px',
+      backgroundColor: '#00689F',
+      fontFamily: 'Montserrat',
+      fontSize: '22px',
+      fontWeight: 500,
+      color: '#FFFFFF',
+      padding: '0 5rem',
+      margin: '2rem auto',
+      height: '50px',
```

Comment:
nit: (optional) the buttons seems a bit wide (a lot of extra empty space beyond the text horizontally), but if you like how it looks you don't need to change it!

Reply:
I do kind of like the wider buttons